server/odc-service/src/main/java/com/oceanbase/odc/service/flow/FlowTaskInstanceLoggerService.java

```diff
@@ -0,0 +1,118 @@
+            TaskService taskService,
+            LoggerProperty loggerProperty) {
+        this.flowTaskInstanceService = flowTaskInstanceService;
+        this.requestDispatcher = requestDispatcher;
+        this.dispatchChecker = dispatchChecker;
+        this.taskService = taskService;
+        this.loggerProperty = loggerProperty;
+    }
+
+    @SneakyThrows
+    public String getLog(OdcTaskLogLevel level, Long flowInstanceId, boolean skipAuth) {
+        Optional<TaskEntity> taskEntityOptional =
+                flowTaskInstanceService.getLogDownloadableTaskEntity(flowInstanceId, skipAuth);
+        if (!taskEntityOptional.isPresent()) {
+            log.warn("get log failed, flowInstanceId: {}, skipAuth: {}", flowInstanceId, skipAuth);
+            return String.format("get log failed, id=%s", flowInstanceId);
+        }
+        TaskEntity taskEntity = taskEntityOptional.get();
+        if (!dispatchChecker.isTaskEntityOnThisMachine(taskEntity)) {
+            ExecutorInfo executorInfo = JsonUtils.fromJson(taskEntity.getExecutor(), ExecutorInfo.class);
+            DispatchResponse response = requestDispatcher.forward(executorInfo.getHost(), executorInfo.getPort());
+            return response.getContentByType(new TypeReference<SuccessResponse<String>>() {}).getData();
+        }
+        return getLog(taskEntity.getCreatorId(), taskEntity.getId() + "", taskEntity.getTaskType(), level);
+    }
+
+    @SneakyThrows
+    public File downloadLog(Long flowInstanceId, boolean skipAuth) {
+        Optional<TaskEntity> taskEntityOptional =
+                flowTaskInstanceService.getLogDownloadableTaskEntity(flowInstanceId, skipAuth);
+        if (!taskEntityOptional.isPresent()) {
+            throw new NotFoundException(ErrorCodes.NotFound, new Object[] {flowInstanceId},
+                    ErrorCodes.TaskLogNotFound.getLocalizedMessage(new Object[] {"Id", flowInstanceId}));
+        }
+        TaskEntity taskEntity = taskEntityOptional.get();
+        if (!dispatchChecker.isTaskEntityOnThisMachine(taskEntity)) {
+            ExecutorInfo executorInfo = JsonUtils.fromJson(taskEntity.getExecutor(), ExecutorInfo.class);
+            DispatchResponse response = requestDispatcher.forward(executorInfo.getHost(), executorInfo.getPort());
+            return response.getContentByType(new TypeReference<SuccessResponse<File>>() {}).getData();
+        }
+        return getLogFile(taskEntity.getCreatorId(), taskEntity.getId() + "", taskEntity.getTaskType(),
+                OdcTaskLogLevel.ALL);
+    }
+
+    public File getLogFile(Long userId, String flowInstanceId, TaskType type, OdcTaskLogLevel logLevel) {
+        String logFilePath = taskService.getLogFilePath(userId, flowInstanceId, type, logLevel);
+        try {
+            return taskService.getLogFile(logFilePath);
+        } catch (NotFoundException ex) {
+            log.warn(ErrorCodes.TaskLogNotFound.getLocalizedMessage(new Object[] {"Id", flowInstanceId}));
```

Comment:
i18n is not required for log output, use english instead

Reply:
ok